src/utils.ts

```diff
@@ -16,7 +16,7 @@ export function evalTemplate(entity: string | null, template: string, hass: Home
     return new Function('hass', 'state', 'user', 'html', `'use strict'; ${func}`).call(
       null,
       hass,
-      entity != null ? hass.states[entity].state : null,
+      entity != null && entity != undefined ? hass.states[entity].state : null,
```

Comment:
Normally I would suggest using strict quality (`===` and `!==` in JavaScript) here. In this case, loose equality (`==` and `!=`) works in your favor. You have just use `entity != null` and it should catch both the null and undefined cases.

Reply:
Yeah, makes sense; there is a test that covers both